src/modules/song/thunks.ts

```diff
@@ -83,8 +81,8 @@ export const uploadSong = createAsyncThunk(
         body: JSON.stringify({ file: audioBinaryStr }),
       });
 
-      // fetch user's songs
-      dispatch(songApi.endpoints.getSongs.initiate());
+      // navigate to library page to view new song
+      history.push("/home/library");
```

Comment:
lol i actually did this on my local branch as well, minus the comment. 
We are doing a getSongs fetch somewhere else too and i took that off, i'll explain in my PR but this change should be good